DisplayCAL/xh_filebrowsebutton.py

```diff
@@ -43,6 +43,10 @@ def DoCreateResource(self):
             name=self.GetName(),
         )
         self.SetupWindow(w)
+        if self.GetBool("hidden") and w.Shown:
```

Comment:
this was the cause of the extra dropdown which should have been hidden, I do not know wxPython at all and can find v few resources on it & could not work out why SetupWindow was no longer hiding the control as it does on the python2  build so added this workaround here + to all similar custom controls. If anyone knows wxPython and can say how to actually fix the problem, happy to change htis.

Reply:
I'm not experienced in wxpython at all, however it looks a bit strange, like if an argument is not considered anymore.
>SetupWindow was no longer hiding the control as it does on the python2 

it is very likely that this has more to do with the recent wxpython version we are using than with python3 I think

Reply:
it's what I thought also (re wxpython), couldn't find anything in https://wxpython.org/pages/changes/index.html though and there seems relatively little info on wxpython around so workaround seemed fair to me